vm/actor_interface/src/lib.rs

```diff
@@ -3,4 +3,25 @@
 
 mod builtin;
 
+use cid::Cid;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::from_slice;
+use serde::de::DeserializeOwned;
+
 pub use self::builtin::*;
+
+/// Extension methods for inserting and retrieving IPLD data with CIDs
+pub trait BlockstoreExt: Blockstore {
```

Comment:
`BlockstoreExt` does not belong here. You _could_ make it part of the library but it definitely shouldn't be public. I think the best solution would be to not include it at all.

Reply:
removed